dts/bindings/dma/dma-controller.yaml

```diff
@@ -33,3 +33,13 @@ properties:
       type: int
       required: false
       description: Number of DMA request signals supported by the controller.
+
+    dma-offset:
```

Comment:
Since it is stm32 specific, it has to be moved to `dts/bindings/dma/st,stm32-dma.yaml`